### ISSUE TITLE
Git Branch Exercise

### ISSUE THREAD

**Issue by arhenry2** [arhenry2/CompToolsNotes#2]
# Git Branch Exercise

Link to branch network view: https://github.com/arhenry2/CompToolsNotes/network

**Note**: I, Ashley, named my Branch A "quercus" to be fun, but realized now it could be confusing. Adam correctly named Branch B "B". I apologize for any confusion!

Thank you,
Ashley Henry and Adam Bigott

@adambigott 
@cecileane
@coraallencoleman

**Comment by cecileane** [comment]
I like quercus!
I don't see any commit from Adam, though.

**Comment by arhenry2** [comment]
Good catch! We'll redo the exercise and I'll send a new issue with the updated branch view.
